hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/routes.tsx

```diff
@@ -58,6 +59,10 @@ export const routes: IRoute[] = [
     path: '/Containers',
     component: MissingContainers,
   },
+  {
+    path: '/HeatMap',
+    component: HeatMap
```

Comment:
if you want to do the rename in this PR
```suggestion
    path: '/Heatmap',
    component: Heatmap
```